Assets/HoloToolkit-Examples/Input/Scenes/BasicCursor.unity

```diff
@@ -308,7 +304,7 @@ Prefab:
         type: 2}
       propertyPath: Cursor
       value: 
-      objectReference: {fileID: 531712469}
+      objectReference: {fileID: 0}
```

Comment:
This reference should be replaced with the new cursor on Managers -> InputManager's SimpleSinglePointerSelector script.